Source/Magick.NET/Shared/Optimizers/ImageOptimizer.cs

```diff
@@ -35,6 +35,15 @@ public bool OptimalCompression
             set;
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to skip unsupported files instead of throwing an exception.
+        /// </summary>
+        public bool IgnoreUnsupportedFormats
```

Comment:
The properties and methods are sorted alphabetically, could you move this?

Reply:
Yes the properties are now in correct order